components/VillagerCard.jsx

```diff
@@ -1,17 +1,48 @@
 import StyledImage from '../components-styled/StyledImage';
 import StyledCard from '../components-styled/StyledCard';
+import { useState } from 'react';
+import StyledOverlay from '../components-styled/StyledOverlay';
+import VillagerDetailedCard from './VillagerDetailedCard';
 
-export default function VillagersCard({
+export default function VillagerCard({
   name,
   personality,
   species,
   gender,
   imageUri,
   index,
+  birthday,
+  catchPhrase,
+  bubbleColor,
+  textColor,
```

Comment:
Da es langsam sehr viele Props werden, könnte man überlegen, ob man nicht doch den gesamten array eintrag übergibt und sich dann hier destructured was man braucht und auch den gemsamten eintrag in die Detailcard reicht. Also ich meine das gesamte Element aus dem Map.